test/test_eos.py

```diff
@@ -0,0 +1,114 @@
+from __future__ import division, absolute_import, print_function
+
+__copyright__ = (
+    """Copyright (C) 2020 University of Illinois Board of Trustees"""
+)
+
+__license__ = """
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+"""
+
+import numpy as np
+import numpy.linalg as la  # noqa
+import pyopencl as cl
+import pyopencl.clrandom
+import pyopencl.clmath
+from meshmode.mesh import BTAG_ALL, BTAG_NONE  # noqa
+
+from mirgecom.eos import IdealSingleGas
+from mirgecom.initializers import Vortex2D
+from mirgecom.initializers import Lump
+from grudge.eager import EagerDGDiscretization
+from pyopencl.tools import (  # noqa
+    pytest_generate_tests_for_pyopencl as pytest_generate_tests,
+)
+
+
+def test_idealsingle_lump():
```

Comment:
```suggestion
def test_idealsingle_lump():
    "Tests that the lump IC satisfies ... (what)?"
```
(IMO every test should have a short description like that.)